typescript/conditions.d.ts

```diff
@@ -44,8 +44,17 @@ export type ConditionalPredicateBetween = ConditionalPredicateBase & {
   max: number
 }
 
+export type ConditionalPredicateRepeatableSet = ConditionalPredicateBase & {
+  type: 'REPEATABLESET',
+  predicate: ConditionalPredicateNumeric
```

Comment:
```suggestion
  predicate: Exclude<ConditionalPredicate, ConditionalPredicateRepeatableSet>
```
might be better for if in the unlikely event a new predicate is added?